sample/build.gradle

```diff
@@ -4,41 +4,64 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'uk.gov.hmrc.spotless'
 apply plugin: 'io.gitlab.arturbosch.detekt'
 
+apply from: "./screenshots.gradle"
+
 detekt {
     config = files("detekt/config.yml")
     baseline = file("detekt/baseline.xml")
 }
 
 android {
+    compileSdkVersion compile_sdk_version
+    buildToolsVersion "30.0.3"
```

Comment:
Could this be extracted out so all modules use the same build tools version?

Reply:
Addressed in 694d064736478f073fa510eb663c87a79120f863